.github/workflows/workflow.yml

```diff
@@ -160,8 +167,18 @@ jobs:
           GHCVER="$(ghc --numeric-version)"
           echo "CABALVER=${CABALVER}" >> "${GITHUB_ENV}"
           echo "GHCVER=${GHCVER}"     >> "${GITHUB_ENV}"
+          if [[ "${{ steps.setup.outputs.ghc-version }}" == "latest-nightly" ]]
+          then
+            # TODO: Shouldn't we use steps.setup.outputs.ghcup-release-channel?
+            #       Why is empty?
```

Comment:
`ghcup-release-channel` is not an output of the setup action, you can see inputs and outputs of this action here: https://github.com/haskell/actions/blob/fc675398fab930927d01e9c00eee89066aa2ca12/setup/action.yml

Reply:
I see. So the current use of `matrix.ghcup_release_channel` is OK I guess?